pkg/orchestrator/gitHubActions.go

```diff
@@ -174,18 +174,23 @@ func (g *GitHubActionsConfigProvider) GetReference() string {
 	return getEnv("GITHUB_REF", "n/a")
 }
 
-// GetBuildURL returns the builds URL. For example, https://github.com/SAP/jenkins-library/actions/runs/5815297487
+// GetBuildURL returns the builds URL. The URL should point to the pipeline (not to the stage)
+// that is currently being executed. For example, https://github.com/SAP/jenkins-library/actions/runs/5815297487
 func (g *GitHubActionsConfigProvider) GetBuildURL() string {
 	return g.GetRepoURL() + "/actions/runs/" + g.GetBuildID()
 }
 
-// GetJobURL returns the current job HTML URL (not API URL).
-// For example, https://github.com/SAP/jenkins-library/actions/runs/123456/jobs/7654321
+// GetJobURL returns the job URL. The URL should point to project’s pipelines.
+// For example, https://github.com/SAP/jenkins-library/actions/workflows/workflow-file-name.yaml
 func (g *GitHubActionsConfigProvider) GetJobURL() string {
-	// We need to query the GitHub API here because the environment variable GITHUB_JOB returns
-	// the name of the job, not a numeric ID (which we need to form the URL)
-	g.guessCurrentJob()
-	return g.currentJob.HtmlURL
+	fileName := workflowFileName()
+	if fileName == "" {
+		return ""
```

Comment:
if there is something wrong with GITHUB_WORKFLOW_REF, the GetJobURL returns an empty string, then the telemetry package handles it and put `n/a` to the `PipelineURLHash`